suitcase/specfile/__init__.py

```diff
@@ -277,7 +277,7 @@ def to_spec_scan_data(start, primary_descriptor, event):
 }
 
 
-def export(gen, directory, file_prefix='{uid}-', **kwargs):
+def export(gen, directory, file_prefix='{start[uid]}-', **kwargs):
```

Comment:
I think there should be no trailing `-` here. The `Serializer` does not have one, so this was likely just an accidental mistake in my original implementation. Mind fixing that while you are here?

Reply:
done